pgdb/handler.go

```diff
@@ -0,0 +1,74 @@
+package pgdb
+
+import (
+	"app/config"
+	"app/handlers"
+	"net/http"
+
+	"github.com/jmoiron/sqlx"
+	"github.com/labstack/echo/v4"
+)
+
+// UpsertRasModel ...
+func UpsertRasModel(ac *config.APIConfig, db *sqlx.DB) echo.HandlerFunc {
+	return func(c echo.Context) error {
+
+		definitionFile := c.QueryParam("definition_file")
+
+		if definitionFile == "" {
+			return c.JSON(http.StatusBadRequest,
+				handlers.SimpleResponse{Status: http.StatusBadRequest,
+					Message: "Missing query parameter: `definition_file`"})
+		}
+
+		err := upsertModelInfo(definitionFile, ac, db)
+		if err != nil {
+			return c.JSON(http.StatusNotAcceptable, err)
+		}
+
+		return c.JSON(http.StatusOK, "Successfully uploaded model information for "+definitionFile)
+	}
+}
+
+// UpsertRasGeometry ...
+func UpsertRasGeometry(ac *config.APIConfig, db *sqlx.DB) echo.HandlerFunc {
+	return func(c echo.Context) error {
+
+		definitionFile := c.QueryParam("definition_file")
+
+		if definitionFile == "" {
+			return c.JSON(http.StatusBadRequest,
+				handlers.SimpleResponse{Status: http.StatusBadRequest,
+					Message: "Missing query parameter: `definition_file`"})
+		}
+
+		err := upsertModelGeometry(definitionFile, ac, db)
+		if err != nil {
+			return c.JSON(http.StatusNotAcceptable, err)
+		}
+
+		return c.JSON(http.StatusOK, "Successfully uploaded model geometry for "+definitionFile)
+	}
+}
+
+// VacuumRasViews ...
+func VacuumRasViews(ac *config.APIConfig, db *sqlx.DB) echo.HandlerFunc {
```

Comment:
If this function doesn't use the APIConfig, lets remove it as an arg

Reply:
Also - does it make sense to have the vacuum and refresh be two separate endpoints?

Reply:
Split these functions as suggested, renamed (simplified) endpoints, updated (*corrected*) function call on `main`.